CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
The 'salary' field is redundant and incorrectly typed as string while 'Salary' already exists as a decimal property. Additionally, leaving TODOs in committed code is discouraged. Remove this field before merging.

```suggestion
// Removed temporary 'salary' field before merge
```